testApi/src/test/resources/test_project/localhost.env.json

```diff
@@ -1,10 +1,11 @@
 {
   "constants": {
     "server": "localhost",
-    "port": "8080",
-    "basic_token": "SecretToken"
+    "port": "9966",
+    "basicToken": "SecretToken",
+    "ownerCellPhoneNumber": "6085551749"
   },
   "properties": {
-    "url": "http://{{ server}}:{{ port }}/restcontroller"
+    "url": "http://{{ server}}:{{ port }}/petclinic"
```

Comment:
why `/petclinic`? looks like a remnant of the OpenAPI Pet Clinic demo.

Reply:
This change has been planned. The Unify "dependency" has been removed as it is not easy to get it running. The pet clinic is easy to get running and it is ready for this kind of usage.